client/src/partials/finance/cashflow/cashflow.html

```diff
@@ -0,0 +1,239 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.FINANCE" | translate }}</li>
+      <li class="title">
+        {{ "CASHFLOW.TITLE" | translate }}
+        <b ng-show="ReportCtrl.state !== 'generate'" class="label label-default">
+          <span class="fa fa-cogs"></span> {{ "REPORT.CONFIGURATION" | translate }}
+        </b>
+        <b ng-show="ReportCtrl.state === 'generate'" class="label label-info">
+          <span class="fa fa-check-circle-o"></span> {{ "REPORT.GENERATED" | translate }}
+        </b>
+      </li>
+    </ol>
+
+    <div ng-show="ReportCtrl.state === 'generate'" class="toolbar">
+      <div class="toolbar-item btn-group btn-group-sm">
+        <button type="button" class="btn btn-default" ng-click="ReportCtrl.print()">
+          <i class="fa fa-print"></i> {{ 'FORM.BUTTONS.PRINT' | translate }}
+        </button>
+        <button type="button" class="btn btn-default" ng-click="ReportCtrl.reconfigure()">
+          <i class="fa fa-repeat"></i>
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="flex-content">
+  <div class="container-fluid">
+
+    <div ng-switch="ReportCtrl.state">
+      <div class="row" ng-switch-default>
+
+        <!-- configuration panel  -->
+        <div class="col-xs-6 col-xs-offset-3 panel panel-default">
```

Comment:
Why `xs` and not `md` as usual?  Is there a particular reason for this?

I would also separate the column definitions from the panel definition.  It might lead to unexpected behavior as is.  Try this:
```html
<div class="col-xs-6 ...">
  <div class="panel ...">
    <!-- etc ... -->
  </div>
</div>
```